src/qt/walletcontroller.cpp

```diff
@@ -75,7 +75,7 @@ void WalletController::closeWallet(WalletModel* wallet_model, QWidget* parent)
 {
     QMessageBox box(parent);
     box.setWindowTitle(tr("Close wallet"));
-    box.setText(tr("Are you sure you wish to close wallet <i>%1</i>?").arg(wallet_model->getDisplayName()));
+    box.setText(tr("Are you sure you wish to close the wallet <i>%1</i>?").arg(wallet_model->getDisplayName()));
```

Comment:
This PR and #16826 both touch this one line. Trivial rebase for the whichever is not merged first.